Automattic/jetpack
# Remove JSON polyfill from postmessage

This polyfill hasn't been needed since IE 8 became widely adopted, sometime around the Early Cretaceous Period. Removing it saves more than a few bytes.

Related wpcom change: D54236-code

#### Changes proposed in this Pull Request:
* Remove JSON polyfill from `postmessage.js`

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Go to any site that uses `postmessage`, e.g. `https://automattic.com`
* Ensure that there are no errors

#### Proposed changelog entry for your changes:
* Includes: removed JSON polyfill from `postmessage.js`
